powerauth-java-cmd/src/main/java/io/getlime/security/powerauth/app/cmd/Application.java

```diff
@@ -168,6 +169,13 @@ public void checkServerTrusted(java.security.cert.X509Certificate[] certs, Strin
 
             }
 
+            String deviceInfo;
+            if (cmd.hasOption("D")) {
+                deviceInfo = cmd.getOptionValue("D");
+            } else {
+                deviceInfo = "cmd-tool";
+            }
+
```

Comment:
I would also allow to configure the platform. See my comment in the restful integration PR.

Reply:
Yes, I will add it.